code/ATMOSPHERICS/components/unary_devices/cryo.dm

```diff
@@ -296,6 +291,7 @@
 		if(occupant.stat == 2 || occupant.health >= 100)  //Why waste energy on dead or healthy people
 			occupant.bodytemperature = T0C
 			return
+		occupant.stat = 1

```

Comment:
does this even work? also defines
im pretty sure humans or carbon mobs handle the cryo by checking if they are inside of it each tick